packages/editor/templates/Coin/README.md

```diff
@@ -236,6 +236,15 @@ function isValidAmount(amount) {
 
 And that's all the code. To see the dapp in action, click _Compile_, then _Deploy_ found under the disclosure triangle of the contract file, then open the _Preview_ tab to see the frontend of the dapp.
 
-## Find out more
+## Next Steps
 
 You can read a full tutorial that accompanies this example dapp, plus many more tutorials, on [kauri.io](https://kauri.io/article/bdd65d6155a74b8aa52672b46b7230a8/v1/a-fullstack-dapp-for-creating-tokens).
+
+Other resources useful to continue your dapp development journey are:
+
+- [Remix](https://remix.ethereum.org/): Ethereum IDE and tools for the web
```

Comment:
"and tools for the web" feels pretty vague. I'm not sure what that means. How about "Ethereum IDE specifically for smart contracts, with in-line compile errors & code auto-complete for Solidity"

Reply:
I took this from the current /build page ;)